README.md

```diff
@@ -1,11 +1,72 @@
 [![Build Status](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client.svg?branch=master)](https://travis-ci.com/SUSE-Enceladus/public-cloud-info-client)
 
-pint
-====
+# Introduction
 
 pint (Public Cloud INformation Tracker) is a command line client to access
 the REST API provided by the SUSE Public Cloud Information Service. The
 information service provides data about the images SUSE publishes in various
 public cloud frameworks (Amazon, Google, Microsoft) as well as the
 servers that make up the SUSE maintained and operated update infrastructure
 in those frameworks.
+
+## Quick Start
+
+There are various ways to install the Pint Client:
+
+### Install the python package using pip
+
+```bash   
+  $ pip install susepubliccloudinfo
+```
+
+### Install On SUSE Linux using rpm package  
```

Comment:
I would recommend moving this to be the first install option, with the caveat that it may not install the latest version.

Also suggest registering the relevant openSUSE or SLE repo from https://build.opensuse.org/project/show/Cloud:Tools to get access to the latest packaged version, though that may come with extra python dependencies.

Reply:
agreed